packages/web3-core-method/README.md

```diff
@@ -43,7 +46,13 @@ myCoolLib.sendTransaction({...}, function(){ ... });
 
 ## Types
 
-All the typescript typings are placed in the types folder.
+All the TypeScript typings are placed in the `types` folder.
 
 [docs]: http://web3js.readthedocs.io/en/1.0/
 [repo]: https://github.com/ethereum/web3.js
+[npm-image]: https://img.shields.io/npm/v/web3-core-method.svg
+[npm-url]: https://npmjs.org/package/web3-core-method
+[deps-image]: https://david-dm.org/ethereum/web3.js/1.x/status.svg?path=packages/web3
```

Comment:
Path seems not to be correct here.

Reply:
I sometimes get some request errors on badges, which work after a page refresh. Can it be the case here?

Reply:
Got this as well, I would assume that this is the first image generation trigger since the image was never requested before. Not 100% sure though, might be worth to have a second or third try some time after.

Reply:
@holgerd77 thanks for catching this, fixed.
@evertonfraga yeah it'll have to do the initial request first for some of the packages :)